packages/react-dev-utils/README.md

```diff
@@ -166,6 +166,43 @@ compiler.plugin('done', function(stats) {
 });
 ```
 
+#### `formatter(results: Object): string`
+
+This is an eslint formatter that takes the result generated from eslint and formats the output string
+
+```js
+const formatter = require('react-dev-utils/formatter');
+const webpack = require('webpack');
+
+module: {
```

Comment:
It's not very clear this is Webpack config. I'd prefer that we show a specific ESLint rule alone, with a comment like

```
// ESLint loader in your webpack config:
```